cypress/integration/v3/blocks/column/index.js

```diff
@@ -0,0 +1,64 @@
+
+/**
+ * External dependencies
+ */
+import {
+	assertBlockExist, blockErrorTest,
+} from '~stackable-e2e/helpers'
+import { stkBlocks } from '~stackable-e2e/config'
+
+export {
+	blockExist,
+	blockError,
+	innerBlocks,
+	assertLinking,
+	assertWidth,
+}
+
+function blockExist() {
+	it( 'should show the block', assertBlockExist( 'stackable/columns', '.stk-block-column', { variation: 'Two columns; equal split' } ) )
+}
+
+function blockError() {
+	it( 'should not trigger block error when refreshing the page', blockErrorTest( 'stackable/columns', { variation: 'Two columns; equal split' } ) )
+}
+
+function innerBlocks() {
+	it( 'should allow adding inner blocks', () => {
+		cy.setupWP()
+		cy.newPage()
+		cy.addBlock( 'stackable/columns', { variation: 'Two columns; equal split' } )
+
+		stkBlocks
+			.filter( blockName => blockName !== 'stackable/column' )
+			.forEach( blockName => cy.addInnerBlock( 'stackable/column', blockName, 0 ) )
+
+		cy.savePost()
+	} )
+}
+
+function assertLinking() {
+	it( 'should toggle the linking of the column block', () => {
+		cy.setupWP()
+		cy.newPage()
+		cy.addBlock( 'stackable/columns', { variation: 'Two columns; equal split' } )
+		cy.selectBlock( 'stackable/column', 0 ).toggleBlockLinking( true )
+		cy.selectBlock( 'stackable/column', 1 ).toggleBlockLinking( false )
+		cy.selectBlock( 'stackable/column', 1 ).find( '.stk-linking-wrapper > .stk--is-unlinked' ).should( 'exist' )
```

Comment:
It would be better if we actually test the actual linking here.

Otherwise, we can just remove this and put this inside v3 features folder